bagit.py

```diff
@@ -752,7 +757,7 @@ def _make_manifest(manifest_file, data_dir, processes, algorithm='md5'):
     else:
         checksums = [manifest_line(i) for i in _walk(data_dir)]
 
-    with open(manifest_file, 'w') as manifest:
+    with open_text_file(manifest_file, 'w') as manifest:
```

Comment:
I just pushed out a commit which changes this but got distracted trying to figure out why two separate tests are failing only when run inside a Docker container.